core/app/models/spree/tax_calculator/default.rb

```diff
@@ -0,0 +1,46 @@
+module Spree
+  module TaxCalculator
+    class Default
+      include Spree::Tax::TaxHelpers
+
+      def initialize(order)
+        @order = order
+      end
+
+      def calculate
+        Spree::Tax::Output::Order.new(
+          id: order.id,
+          line_items: line_item_rates,
+          shipments: shipment_rates
+        )
+      end
+
+      private
+
+      attr_reader :order
+
+      def line_item_rates
+        order.line_items.flat_map do |line_item|
+          calculate_rates(line_item)
+        end
+      end
+
+      def shipment_rates
+        order.shipments.flat_map do |shipment|
+          calculate_rates(shipment)
+        end
+      end
+
+      def calculate_rates(item)
+        rates_for_item(item).map do |rate|
+          Spree::Tax::Output::Item.new(
+            id: item.id,
+            tax_rate: rate,
+            amount: rate.compute_amount(item),
+            included_in_price: rate.included_in_price
```

Comment:
This should have the `label` thing as well. We want to abstract from the `Spree::TaxRate`, so `rate` should not be in here.